tests/automation.js

```diff
@@ -21,6 +21,7 @@ casper.options.onWaitTimeout = function() {
 var gfxTests = [
   { name: "gfx/AlertTest", maxDifferent: 1942 },
   { name: "gfx/CanvasTest", maxDifferent: 0 },
+  { name: "gfx/CanvasWithHeaderTest", maxDifferent: 2000 },
```

Comment:
Does this really need to be as high as 2000? I see 1368 different pixels in my runs on Mac, which typically has the highest number of different pixels.